wordpress-mobile/WordPress-iOS
# Confirm that site is indeed not a WordPress site after XML-RPC validation fails

Partially addresses #18497 
Related WordPressAuthenticator PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/796
Related WordPressKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/655

## To test:

A URL for testing can be found at pbArwn-4xZ-p2#comment-5712 and/or 5170539-zd-a8c (not included in this ticket to protect user privacy):

1. Navigate to the app's login screen and select the "Log in by entering site address" option.
2. Submit the test URL.
3. Confirm that XML-RPC error is shown instead of invalid WordPress site error

### Regression

- Valid site scenario
- Running through a few cases described in https://github.com/wordpress-mobile/WordPress-iOS/pull/17724

## Regression Notes
1. Potential unintended areas of impact

Breaking site address login

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual testing, existing tests

3. What automated tests I added (or what prevented me from doing so)

Added new tests into WordPressAuthenticator-iOS

PR submission checklist:


UI Changes testing checklist:
